bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/DataLayerConfig.java

```diff
@@ -39,4 +41,29 @@
     @Property(label="Data Layer client library not included")
     boolean skipClientlibInclude() default false;
 
+    /**
+     *
+     * @return the name of the datalayer
+     * Defaults to adobeDataLayer.
+     */
+    @Property(label = "Data Layer object name")
+    String name() default DATALAYER_OBJECT_NAME_ADOBE;
+
+    /**
+     *
+     * @return the type of push function (push() or gtag()) of the datalayer
+     * Defaults to push.
+     */
+    @Property(label = "Use gtag function to push data")
+    boolean pushFunctionUseGtag() default false;
+
+    /**
+     *
+     * @return ACDL library not included
+     * Defaults to false.
+     */
+    @Property(label = "ACDL library not included")
+    boolean skipAcdlInclude() default false;
```

Comment:
We already have `skipClientlibInclude` that serves the same purpose (to control if the ACDL scripts should be included).

Reply:
see reply to Paga.java above